PyInstaller/building/splash.py

```diff
@@ -389,60 +387,71 @@ def _resize_image(_image, _orig_size):
+            )
 
         image_file.close()
 
-        res = SplashWriter(self.name,  # noqa: F841
-                           self.splash_requirements,
-                           self.tcl_lib[0],  # tcl86t.dll
-                           self.tk_lib[0],  # tk86t.dll
-                           TK_ROOTNAME,
-                           self.rundir,
-                           image,
-                           self.script)
+        SplashWriter(
+            self.name,  # noqa: F841
+            self.splash_requirements,
+            self.tcl_lib[0],  # tcl86t.dll
+            self.tk_lib[0],  # tk86t.dll
+            TK_ROOTNAME,
+            self.rundir,
+            image,
+            self.script
+        )
 
     def test_tk_version(self):
         tcl_version = float(self._tkinter_module.TCL_VERSION)
         tk_version = float(self._tkinter_module.TK_VERSION)
 
         # Test if tcl/tk version is supported
         if tcl_version < 8.6 or tk_version < 8.6:
-            logger.warning("The installed Tcl/Tk (%s/%s) version might not"
-                           " work with the splash screen feature of the"
-                           " bootloader. The bootloader is tested against"
-                           " Tcl/Tk 8.6" % (self._tkinter_module.TCL_VERSION,
-                                            self._tkinter_module.TK_VERSION))
+            logger.warning(
+                "The installed Tcl/Tk (%s/%s) version might not"
+                " work with the splash screen feature of the"
+                " bootloader. The bootloader is tested against"
+                " Tcl/Tk 8.6" % (self._tkinter_module.TCL_VERSION, self._tkinter_module.TK_VERSION)
+            )
 
         # This should be impossible, since tcl/tk is released together with
         # the same version number, but just in case
         if tcl_version != tk_version:
-            logger.warning("The installed version of Tcl (%s) and Tk (%s) do"
-                           " not match. PyInstaller is tested against matching"
-                           " versions" % (self._tkinter_module.TCL_VERSION,
-                                          self._tkinter_module.TK_VERSION))
+            logger.warning(
+                "The installed version of Tcl (%s) and Tk (%s) do"
+                " not match. PyInstaller is tested against matching"
+                " versions" % (self._tkinter_module.TCL_VERSION, self._tkinter_module.TK_VERSION)
+            )
 
         # Test if tcl is threaded.
         # If the variable tcl_platform(threaded) exist, the tcl
         # interpreter was compiled with thread support.
-        threaded = bool(exec_statement("""
+        threaded = bool(
+            exec_statement(
+                """
         from tkinter import Tcl, TclError
         try:
             print(Tcl().getvar('tcl_platform(threaded)'))
         except TclError:
-            pass"""))
+            pass"""
+            )
```

Comment:
We might want to adjust indentation of embedded code here, to match our usual style (unless yapf would undo that).